### ISSUE TITLE
Bug when displaying expression errors in multiline scripts

### ISSUE THREAD

**Issue by hugo-syn** [rhysd/actionlint#330]
_actionlint_ fails to display the position of the error when using multiline scripts in the _rule_expression.go_ like this one:

```yaml
name: Test

on: push
jobs:
  dummyjob:
    runs-on: ubuntu-latest
    steps:
      - name: shell
        run: |
        
            echo '${{ toJSON(github.event.issue.body) }}'
```

```
actionlint test/test2.yml 
test/test2.yml:9:32: "github.event.issue.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details [expression]
```

I looked at the code, but there are a lot of corner cases :/
